pkg/accounting/tally/tally.go

```diff
@@ -26,24 +26,38 @@ type Tally interface {
 }
 
 type tally struct {
-	pointerdb *pointerdb.Server
-	overlay   pb.OverlayServer
-	kademlia  *kademlia.Kademlia
-	limit     int
-	logger    *zap.Logger
-	ticker    *time.Ticker
-	db        *dbx.DB
+	pointerdb  *pointerdb.Server
+	overlay    pb.OverlayServer
+	kademlia   *kademlia.Kademlia
+	limit      int
+	logger     *zap.Logger
+	ticker     *time.Ticker
+	nodes      map[string]int64
```

Comment:
I'm thinking that this shouldn't be part of tally, it will persist between identifyActiveNodes()'s